src/gradients-schemes/ScaleScheme.java

```diff
@@ -67,14 +67,15 @@ else if ( var > min )
 			{
 				double tempval = min - var ;
 				double tempmax = min - max;
-				perc = tempval / tempmax ;
+        tempmax /= size;
```

Comment:
make comment explaining that (min - max)/size is the width of each color slice, and that